agent/lib_aws_sdk_php.h

```diff
@@ -0,0 +1,12 @@
+/*
+ * Copyright 2024 New Relic Corporation. All rights reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * Functions relating to instrumenting AWS-SDK-PHP.
+ */
+#ifndef LIB_AWS_SDK_PHP_HDR
+#define LIB_AWS_SDK_PHP_HDR
+
+extern void nr_aws_sdk_php_enable(TSRMLS_D);
```

Comment:
```suggestion
extern void nr_aws_sdk_php_enable();
```

Reply:
removed in another commit [a36d635](https://github.com/newrelic/newrelic-php-agent/pull/932/commits/a36d635cbabc48c2be756ac0eb3ecfb7682a48bd)